index.html

```diff
@@ -26,6 +26,12 @@ <h1>Pokemon Chart</h1>
 		<td>Fire</td>
 		<td><a href="http://pokemon.wikia.com/wiki/Charmeleon">Charmeleon</a></td>
 	</tr>
+	<tr>
+		<td><img src="https://vignette.wikia.nocookie.net/pokemon/images/4/4a/005Charmeleon.png/revision/latest/scale-to-width-down/350?cb=20140328191309"></td>
+		<td>Charmeleon</td>
+		<td>Fire</>
+		<td><a href="https://pokemon.fandom.com/wiki/Charmeleon">Charmeleon</td>

```

Comment:
This must be link and text for evolution form

Reply:
Fixed